core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -58,7 +58,8 @@ object ConsumerGroupCommand extends Logging {
 
     try {
       if (opts.options.has(opts.listOpt))
-        consumerGroupService.listGroups().foreach(println(_))
+        if (opts.options.has(opts.regexOpt)) consumerGroupService.listGroupsByRegex().foreach(println(_))
```

Comment:
Do we also want to support regex for other commands as well?

Reply:
> Do we also want to support regex for other commands as well?

Not sure, which commands would we like to support?